.github/workflows/check-workflow.yml

```diff
@@ -47,5 +47,5 @@ jobs:
         if ($null -ne $updateResult) {
           $reposWithUpdates = $updateResult[-1]
           Write-Host "Are there any updates available? [$($reposWithUpdates.Count -gt 0)]"
-          echo "::set-output name=update::$($reposWithUpdates.Count -gt 0)"
+          echo "name=update::$($reposWithUpdates.Count -gt 0)" >> $env:$GITHUB_OUTPUT
```

Comment:
The name of the variable part can now be skipped, so it will just need to be outputted like this:
`variable-name=value`

```suggestion
          echo "update=$($reposWithUpdates.Count -gt 0)" >> $env:$GITHUB_OUTPUT
```